M_Rizki_Maulana_211011400080/tugas2_M_Rizki_Maulana_211011400080.docx

```diff
@@ -0,0 +1,17 @@
+Fungsional
+-Dimeja 
+A. Pelanggan harus melihat detail item dan menambahkan produk ke makananan
+B. Sistem akan menampilkan kategori makanan dan item dalam kartu makanan
+C. Sistem akan melacak pesanan langsung dan pembaruan status untuk pengiriman makanan
+D. Sistem akan menambahkan perkiraan waktu pengiriman untuk pesanan
+C. Sistem akan merubah setiap bahan makanan itu habis
+
+- Chef
+A. Koki dapat melihat resep menu makanan dan minuman
+B. Koki dapat mencentrang menu yang sudah dibuat atau sudah selesai makanan
+C. Koki dapat memperbaharui bahan makanan yang sedia atau tidak tersedia
+
+Non Fungsional
+A. Pemesanan menu dilakukan dengan cepat, akurat dan terpecaya
+B. Sistem seharusnya dapat diedit setelah sekian lama dengan mudah
+C. Pengguna dapat memesan makanannya dalam waktu 7 menit
```

Comment:
Great, but you can do more than this. Analisis requirement itu harus dibuat sejelas-jelasnya agar tidak terjadi kesalahpahaman antara software yang dibuat dengan keinginan user.